lib/data/2024_hud_lists.json

```diff
@@ -2825,7 +2825,7 @@
       },
       {
         "key": 436,
-        "description": "Emergency Housing Voucher"
+        "description": "Housing Stability Voucher"
```

Comment:
renamed in 2024 spec, we missed it
<img width="725" alt="Screenshot 2024-05-02 at 3 35 50 PM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/b8d6d233-e123-4b03-9e04-38b19e1bd2bf">